From 0908d9ef85528fb32f0456b3e1cd91f3e2b84cfd Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@arm.com>
Date: Fri, 20 Aug 2021 08:47:12 -0500
Subject: [PATCH 35/37] uefi-sct/SctPkg: Secure Boot: increase future date
 variable to 5 yrs

Secure boot variables are time based-- any update to a variable must have
a timestamp date later than the variable being updated.  In order to be able
to delete PK/KEK/db/dbx create null siglist with a date 5 years in the future.

Signed-off-by: Stuart Yoder <stuart.yoder@arm.com>
---
 .../SecureBoot/BlackBoxTest/Dependency/Images/GNUmakefile       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/Dependency/Images/GNUmakefile b/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/Dependency/Images/GNUmakefile
index f3208129..e4edb2d0 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/Dependency/Images/GNUmakefile
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/Dependency/Images/GNUmakefile
@@ -28,7 +28,7 @@
 BASE_NAME=SecureBoot
 TARGET=$(BIN_DIR)/$(BASE_NAME)
 SOURCE_DIR=$(WORKSPACE)/SctPkg/TestCase/UEFI/EFI/RuntimeServices/SecureBoot/BlackBoxTest/Dependency/Images
-FUTURE_DATE=$(shell date --rfc-3339=date -d "+1 year")
+FUTURE_DATE=$(shell date --rfc-3339=date -d "+5 year")
 FUTURE_DATE2=$(shell date --rfc-3339=date -d "+1 day")
 FUTURE_DATE3=$(shell date --rfc-3339=date -d "+2 day")
 FUTURE_DATE4=$(shell date --rfc-3339=date -d "+3 day")
-- 
2.17.1

